qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -45,6 +48,27 @@ def change_virtio_media(cdrom_virtio):
         test.log.info("Changing virtio iso image to '%s'", virtio_iso)
         vm.change_media("drive_virtio", virtio_iso)
 
+    def check_network_config(session_serial):
```

Comment:
Doc string please.

Reply:
done

Reply:
Please follow https://peps.python.org/pep-0257/

Reply:
Sorry, I got the quotes wrong. I have read the PEP standard and got the code to be correct.